src/Administration/Resources/app/administration/src/app/component/entity/sw-one-to-many-grid/index.js

```diff
@@ -63,18 +63,33 @@ Component.extend('sw-one-to-many-grid', 'sw-data-grid', {
             page: 1,
             limit: 25,
             total: 0,
+            /* @deprecated: tag:v6.7.0 - Will be removed because wrong typo, use initial instead */
             intial: true,
+            initial: true,
         };
     },
 
+    watch: {
+        collection: {
+            handler() {
+                if (!this.initial) {
+                    this.load();
+                }
+            },
+            deep: true,
+        },
+    },
+
     methods: {
         createdComponent() {
             this.$super('createdComponent');
 
             // assign collection as records for the sw-data-grid
             this.applyResult(this.collection);
 
-            this.intial = false;
+            /* @deprecated: tag:v6.7.0 - Will be removed */
+            this.initial = this.intial;
+            this.initial = false;
```

Comment:
From line number 90 to 92. This handling is incorrect. I'd only do `this.initial = this.intial;` and replace `intial` by `initial` at anywhere it represents.

Reply:
Suggestion:

```ts
            /* @deprecated: tag:v6.7.0 - Will be removed */
            this.intial = false;
           
            this.initial = false;
```